README.md

```diff
@@ -1,216 +1,265 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Mobile Application
+# Compose Multiplatform mobile application
 
-Use this template to start developing your own Compose Multiplatform application targeting Android and iOS (Alpha).
+> Compose Multiplatform is in Alpha. It may change incompatibly and require manual migration in the future.
+
+You can use this template to start developing your own [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) application.
+The result will be a Kotlin Multiplatform project targeting Android and iOS.
```

Comment:
I think that it would be helpful to add a link to the Kotlin Multiplatform landing page or docs.

Reply:
Or maybe we just don't need this sentence at all. What value does it provide to readers? 🤔

Reply:
Don't you think it could be confusing as KMM landing page and docs describe a different technology

Reply:
I am not a fan of this sentence either.

Reply:
I wrote a new intro, but kept the mention of Kotlin Multiplatform, as Seb argued it's worth noting that the project is built on top of the Kotlin Multiplatform